.gitignore

```diff
@@ -76,7 +76,16 @@ islandora_workbench
 jwt
 syn-keys
 codebase/vendor/*
+
+# ignore all the codebase/web folder but don't ignore the custom modules
+!codebase/web/
```

Comment:
I think this is not required, or it will start tracking everything in codebase/web